src/makeHttpRequest.js

```diff
@@ -0,0 +1,76 @@
+/* @flow */
+
+type Options = {
+  method?: string,
+  url: string,
+  params:
+    | string
+    | {
+        [name: string]: string,
+      },
+  headers: Object,
```

Comment:
We need to make `headers` optional so that flow passes. My bad, since I was using always headers on my project, so it should be:

`headers?: Object`

Reply:
same for params ? because it's also optional for me, you want me to put params?: ... ?

Reply:
I tried both of options to test the package but every time I have a problem with react-native dependency required twice (once by my project and once by react-native-offline package).
If i delete node_modules of react-native-offline everything works fine but I cannot use flow :/

```
error: bundling failed: ambiguous resolution: module `myproject/index.js` tries to require `react-native`, but there are several files providing this module. You can delete or fix them: 

  * `myproject/node_modules/react-native/package.json`
  * `route-to-react-native-offline/node_modules/react-native/package.json`
```
Do you have an idea how to fix that ? It's interesting for me to resolve this as I want to work on more packages later.
Thanks

Reply:
OK, only solution I found is to move react-native dependency to peerDependencies.
I will send you this modification also, tell me if you want me to rollback to keep react-native in devDependencies. Anyway I can confirm that I cannot run my app if I leave it as it is